src/api/config/index.ts

```diff
@@ -93,13 +93,19 @@ export interface FeaturedPackagesDetail {
 
 export interface FeatureFlags {}
 
+export interface Feed {
+  type: string;
```

Comment:
Just double checking this is the right field (in the other PR we changed a field named type to mimeType)